panther_battery/README.md

```diff
@@ -19,9 +19,73 @@ Publishes battery state read from ADC unit. Available from Panther version 1.2.
+| Current (charge)                            | `I_charge`                                                    | `I_charge_bat_1 + I_charge_bat_2`                | As above                                                                                         | -            |
+| Percentage                                  | `percentage`                                                  | `(V_bat - V_BAT_MIN) / (V_BAT_FULL - V_BAT_MIN)` | -                                                                                                | `/battery`   |  |
+
+- for double battery:
+
+| Value                       | Variable name                               | Method of calculating                                                       | Published on                           |
+| --------------------------- | ------------------------------------------- | --------------------------------------------------------------------------- | -------------------------------------- |
+| Channel voltage             | `V_bat_1`, `V_bat_2`                        | Measured with ADC unit separately for each battery                          | `/battery_1`, `/battery_2`             |
+| Mean voltage                | `V_bat`                                     | `V_bat_1 + V_bat_2`                                                         | `/battery`                             |
+| Channel temperature         | `temp_bat_1`, `temp_bat_2`                  | Measured with thermistor separately for each battery                        | `/battery_1`, `/battery_2`             |
+| Channel current (discharge) | `I_bat_1`, `I_bat_2`                        | Measured with ADC unit separately for each battery                          | -                                      |
+| Channel current (charge)    | `I_charge_bat_1`, `I_charge_bat_2`          | Measured with ADC unit separately for each battery                          | -                                      |
+| Mean current (discharge)    | `I_bat` *Mean value of two ADC channels*    | `-(I_bat_1 + I_bat_2) + I_charge_bat_1 + I_charge_bat_2,`                   | `/battery`                             |
+| Current (discharge)         | `I_bat`                                     | `-I_bat_? + I_charge_bat_?`                                                 | `/battery_1`, `/battery_2`             |
+| Mean current (charge)       | `I_charge` *Mean value of two ADC channels* | `I_charge_bat_1 + I_charge_bat_2`                                           | -                                      |
+| Current (charge)            | `I_charge`                                  | `I_charge_bat_x`                                                            | -                                      |
```

Comment:
For means there are: V_bat_mean, I_bat_mean variables. I would describe them more like `sum(I_bat)/mean_length` 
Calculating for I_bat and I_charge should be as it is for mean values now.